### DISCUSSION TITLE
Cannot find module '/workspace/.next/server/font-manifest.json'

### DISCUSSION THREAD

**Discussion by ThivaKochasoft** [vercel/next.js#25298] [Help]
### What version of Next.js are you using?

10.2.1

### What version of Node.js are you using?

12

### What browser are you using?

chrome

### What operating system are you using?

GCP app engine

### How are you deploying your application?

clould build

### Describe the Bug

Error: Cannot find module '/workspace/.next/server/font-manifest.json'
Default
2021-05-19 23:07:01.754 IST
Require stack:
Default
2021-05-19 23:07:01.754 IST
- /workspace/node_modules/next/dist/next-server/server/require.js
Default
2021-05-19 23:07:01.754 IST
- /workspace/node_modules/next/dist/next-server/server/load-components.js
Default
2021-05-19 23:07:01.754 IST
- /workspace/node_modules/next/dist/next-server/server/api-utils.js
Default
2021-05-19 23:07:01.754 IST
- /workspace/node_modules/next/dist/next-server/server/next-server.js
Default
2021-05-19 23:07:01.754 IST
- /workspace/node_modules/next/dist/server/next.js

### Expected Behavior

work without an error

### To Reproduce

do a prod build and run

**Comment by anh-duc-le** [comment]
@ThivaKochasoft have you since had any success on getting this to work properly? We're having similar problems in our production deployments

**Reply by anh-duc-le** [reply]
We used the fix discussed in this issue for fixing our production environment. https://github.com/vercel/next.js/issues/24605#issuecomment-830205146

Setting `optimizeFont: false` fixed it for us

**Comment by antonve** [comment]
I was able to fix this by upgrading to Next.js v10.2.2. I was previously on v10.0.6 when the error started appearing in production builds all of a sudden.
